cmd/cogito/main_test.go

```diff
@@ -69,21 +69,25 @@ func TestRunPutSuccess(t *testing.T) {
 	googleChatSpy := testhelp.SpyHttpServer(&chatMsg, chatReply, &gchatUrl, http.StatusOK)
 	in := bytes.NewReader(testhelp.ToJSON(t, cogito.PutRequest{
 		Source: cogito.Source{
-			Owner:        "the-owner",
-			Repo:         "the-repo",
-			AccessToken:  "the-secret",
-			GChatWebHook: googleChatSpy.URL,
-			LogLevel:     "debug",
+			Owner:             "the-owner",
+			Repo:              "the-repo",
+			AccessToken:       "the-secret",
+			GithubApiEndpoint: gitHubSpy.URL,
```

Comment:
Similarly as a previous comment, here we need a name like GitHubApiHostname, without the "endpoint" part.

Reply:
Changed as suggested.